rootfs/etc/cont-init.d/60-svc-rspamd.sh

```diff
@@ -28,3 +28,7 @@ with-contenv
 /usr/sbin/rspamd -i -f -u rspamd -g rspamd
 EOL
 chmod +x /etc/services.d/rspamd/run
+
+# Empty the local_addrs array to avoid having Rspamd skip DMARC and SPF checks if the mailserver is running in a container as well (therefore being in a local network).
+# Required since AnonAddy checks the headers injected by Rspamd. See https://github.com/anonaddy/docker/issues/192#issuecomment-1518111988
+sed -i 's/local_addrs.*$/local_addrs=[]/' /etc/rspamd/options.inc
```

Comment:
Sorry for the delay, can we make this configurable through an env var? Maybe smth like `RSPAMD_NO_LOCAL_ADDRS` which defaults to `false`.

Reply:
I have gone ahead and added an env var